src/resources/material.js

```diff
@@ -238,7 +238,14 @@ class MaterialHandler {
 
             // data[name] contains an asset id for a texture
             // if we have an asset id and nothing is assigned to the texture resource or the placeholder texture is assigned
-            if (data[name] && (!materialAsset.resource[name] || materialAsset.resource[name] === this._getPlaceholderTexture(name, materialAsset))) {
+            // or the data has changed
+            const dataAssetId = data[name];
+
+            const materialTexture = material[name];
```

Comment:
shouldn't this be `... = material.resource[name]`?

Reply:
Line 226 `material = materialAsset.resource`

The old code was `materialAsset.resource[name]`

Reply:
👍🏻